src/test/java/org/takes/facets/flash/RsFlashTest.java

```diff
@@ -63,4 +68,32 @@ public void addsCookieToResponse() throws IOException {
             )
         );
     }
+
+    /**
+     * The expiration date for a cookie must be a GMT time.
```

Comment:
@danff please check put this article about test method names and javadoc 
www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html?expand=1#test-method-names